### ISSUE TITLE
TEST-29-PORTABLE fails due to setfacl on testdir

### ISSUE THREAD

**Issue by bluca** [systemd/systemd#19519]
I've bisected the failure in TEST-29-PORTABLE, and (very weirdly) it looks like it was introduced by PR https://github.com/systemd/systemd/pull/19382 or more precisely by this line:

```
setfacl -d -m "user:${SUDO_USER:?}:r-X" "${TESTDIR:?}"
```

https://github.com/systemd/systemd/blob/main/test/test-functions#L2474

The preceding setfacl doesn't seem to cause any issues. Commenting that one line however, is enough to make the test reliably pass, while it is reliably failing right now. Quite bizzarre.

@keszybz any clue what could be going on? Does the test pass for you as well if that line is removed?

**Comment by yuwata** [comment]
Fix (or workaround) is waiting in #19521.

**Comment by keszybz** [comment]
```
Creating 4.0 filesystem on /var/tmp/systemd-test.U24R3a/root/usr/share/minimal_0.raw, block size 131072.

Unrecognised xattr prefix system.posix_acl_default

Unrecognised xattr prefix system.posix_acl_access

Unrecognised xattr prefix system.posix_acl_access

Unrecognised xattr prefix system.posix_acl_access

Unrecognised xattr prefix system.posix_acl_default

Unrecognised xattr prefix system.posix_acl_access
```
This seems to be caused by lack of support: https://patchwork.kernel.org/project/linux-fsdevel/cover/cover.1548406694.git.geliangtang@gmail.com/, https://github.com/AgentD/squashfs-tools-ng/issues/25#issuecomment-549189690.

What is strange, this causes the permissions on the tree to be different:
```console
$ diffoscope /var/tmp/systemd-test.good/root/usr/share/minimal_0.raw /var/tmp/systemd-test.bad/root/usr/share/minimal_0.raw
--- /var/tmp/systemd-test.good/root/usr/share/minimal_0.raw
+++ /var/tmp/systemd-test.bad/root/usr/share/minimal_0.raw
├── unsquashfs -s {}
│ @@ -1,10 +1,10 @@
│  Found a valid SQUASHFS 4:0 superblock
│ -Creation or last append time Thu May  6 06:44:41 2021
│ -Filesystem size 2865788 bytes (2798.62 Kbytes / 2.73 Mbytes)
│ +Creation or last append time Thu May  6 06:38:55 2021
│ +Filesystem size 2865806 bytes (2798.64 Kbytes / 2.73 Mbytes)
│  Compression gzip
│  Block size 131072
│  Filesystem is exportable via NFS
│  Inodes are compressed
│  Data is compressed
│  Uids/Gids (Id table) are compressed
│  Fragments are compressed
├── unsquashfs -d  -lls {}
│ @@ -1,150 +1,150 @@
│  Parallel unsquashfs: Using 4 processors
│  96 inodes (192 blocks) to write
│  
│ -drwxr-xr-x root/root               393 2021-05-06 06:44 
│ -lrwxrwxrwx root/root                 7 2021-05-06 06:44 /bin -> usr/bin
│ -lrwxrwxrwx root/root                 7 2021-05-06 06:44 /bin_1 -> usr/bin
...
│ +drwxr-x--- root/root               393 2021-05-06 06:38 
│ +lrwxrwxrwx root/root                 7 2021-05-06 06:27 /bin -> usr/bin
│ +lrwxrwxrwx root/root                 7 2021-05-06 06:38 /bin_1 -> usr/bin
```

I think @yuwata's workaround should be good enough.
